SpatialGDK/Source/SpatialGDK/Public/Interop/SpatialReceiver.h

```diff
@@ -152,12 +152,13 @@ class USpatialReceiver : public UObject
 
 	void OnDisconnect(Worker_DisconnectOp& Op);
 
+	void RemoveActor(Worker_EntityId EntityId);
```

Comment:
Don't really want to make this part of the public api.  I could add a RequestRemoveActor or similar.

Reply:
Longer term i'd like to see this method along with others refactored out to a shared location that both the sender and receiver have access to.